src/views/index.html

```diff
@@ -127,12 +127,12 @@
                 flex-grow: 1;
                 display: flex;
                 text-align: center;
-                border: 1px solid #191C22;
+                border: 1px solid rgba(255, 255, 255, 0.2);
```

Comment:
Why use rgba over hex (#FFFFFF33)? I guess it doesn't matter which is used, but it should be standard across the file for consistency. That's something we can do later on.

Reply:
Hex codes don't support transparency. Regardless of if the tab background color changes, this will always make the border "slightly lighter" to stand out.

Reply:
Huh. I thought you could do `#RRGGBBAA`

Reply:
!!! You are right @ABouzo, my mistake. Learn something new every day 😄